test/chunking-form/samples/export-default-global/_expected/es/main.js

```diff
@@ -1,3 +0,0 @@
-console.log(global);
-
-export default global;
```

Comment:
This test case very much should work - I don't think this should be removed.

Reply:
It was not removed, it was just moved elsewhere. But I actually moved it back now.

Reply:
I see, looks good.